browser/src/canvas/sections/CommentSection.ts

```diff
@@ -337,9 +337,10 @@ export class Comment extends CanvasSectionObject {
 			lastPosY = childPositions[i].posY + 24;
 		}
 		if (i < this.sectionProperties.childLines.length) {
-			for (let j = i; j < this.sectionProperties.childLines.length; j++)
+			for (let j = i; j < this.sectionProperties.childLines.length; j++) {
```

Comment:
For what it's worth, I think the failure was this patch exposing this pre-existing bug.